### PR TITLE
147 citation in readme

### LINKED ISSUE #147: Add citation info
Should include a citation for easier referencing, multiple options available. We could link with zenodo for doi minting of new releases or skip version specific DOIs and just go with a good old bites entry.

### PR DESCRIPTION
**Pull request by kkmann**
Closes #147

- [x] add link to pkgdown citation info in readme
- [x] add citation() output to readme

Note, the autogenerated citation info produces a warning at the moment since we do not use a `Date:` field in the DESCRIPTION file. I would not change that and also not add a custom citation to make sure we are always up to date (pun not intended). Will eventually be replaced by reference to journal publication.

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -100,3 +100,10 @@ at:
 ```{r algo-vignette, eval = FALSE}
 vignette("algorithm")
 ```
+
```
Comment (danielinteractive) [inline README.Rmd:103]: Maybe put this under a Citing section or so?

**Review by danielinteractive — COMMENTED** [review]
Thanks @kkmann !

File: README.Rmd (hunk 1 above)
Comment (kkmann) [reply README.Rmd:124]: done

**Review by danielinteractive — APPROVED** [review]

**Comment by kkmann** [comment]
@danielinteractive somehow the checks seem to be pending - any idea?

**Comment by danielinteractive** [comment]
@kkmann not sure what is going on there, let me push an empty commit to restart those

**Comment by danielinteractive** [comment]
@kkmann seems to work now again

**Comment by kkmann** [comment]
🎉

**Pull request merged**